src/styles/1-helpers/_variables.scss

```diff
@@ -0,0 +1,9 @@
+// colors
+$white: #fff;
+$blue: #00acdc;
+$black: #060b35;
+$grey: #616070;
+
+// general
+$fz: 12px;
+$lh: 18px;
```

Comment:
Variable names should be given in such a way that it is clear what they are for in any context.
Therefore, it is not necessary to shorten the names to unreadable ones.